ctapipe/configuration/test/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Please rename the `test` folder to `tests`.
The `__init__.py` file in `tests` can be completely empty ... no need for this license line and docstring.